.github/workflows/lesson_checks.yml

```diff
@@ -0,0 +1,68 @@
+# Contains jobs corresponding to verify Oppia lesson compatibility.
+
+name: Checks for Oppia Lesson Compatibility
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "30 02 * * 1"
+
+jobs:
+  check_lesson_compatibility:
+    name: Check production lesson conversion
+    runs-on: ubuntu-20.04
```

Comment:
Most Web actions are ubuntu-22.04 FYI -- this might become obsolete earlier as time goes on.

Reply:
Our actions are much more sensitive to OS changes than web's. I don't mind testing Ubuntu 22, but we don't take latest generally until we start getting a push to upgrade.

Reply:
Updated to 22.04 and it seems to run fine, which is nice to know. That means there's a good chance the rest of the workflows could be safely updated in the future, as well.

Resolving since you approved and this is a trivial change that was completed.